lib/paperclip/io_adapters/identity_adapter.rb

```diff
@@ -1,12 +1,12 @@
 module Paperclip
   class IdentityAdapter < AbstractAdapter
-    def new(adapter)
-      adapter
```

Comment:
This is very confusing to me. Do we need this identity adapter? Do we need it to have a `new` class or instance method? The current PR changes how the Ruby language works (`.new` initializes and returns an instance), and that is not right. What happens if we drop this adapter?